std/os/index.zig

```diff
@@ -1683,7 +1763,7 @@ test "std.os" {
 
 
 // TODO make this a build variable that you can set
-const unexpected_error_tracing = false;
+const unexpected_error_tracing = true;
```

Comment:
we should turn this off now that we're done with the implementation. It's not meant to be on in master branch.